### ISSUE TITLE
View an activity log

### ISSUE THREAD

**Issue by fricklerhandwerk** [Nix-Security-WG/nix-security-tracker#169]
As a user of the tracker I want to be able to inspect the history of activity on a security issue (or, before a record is issued: a suggestion matching CVEs to packages), including who made a change and when it was made.

For example:
- automatically created matching suggestion
- X selected a suggestions for refinement
- Z created an issue draft from a suggestion
- U dismissed a matching suggestion
- W restored a previously dismissed matching suggestion
- V dismissed a suggestion under refinement

```mermaid
flowchart TD
    A[Matching suggestion created automatically] -->|"Select"| B[Suggestion under refinement]
    A -->|"Dismiss"| C[Dismissed suggestion]
    C -->|"Restore"| B
    B -->|"Create draft"| D[Issue draft]
    B --> |"Dismiss"| C
    
    style A fill:#e1f5fe
    style B fill:#fff3e0
    style C fill:#ffebee
    style D fill:#
```

The above only cover interactions with suggestions, and we currently haven't fully scoped out later workflows. Before an issue is published, the draft may get edited, with various structured modifications For reference, this is what could happen in the future:
- Set a notification interval
- Select/deselect maintainers to notify
- Close a draft unpublished
- Publish an issue from a draft
- Close an issue as resolved
- Send a notification to maintainers
- etc.

Depends on:
- #208

**Comment by fricklerhandwerk** [comment]
Note: We may want the activity log only to show on a suggestion's detail view.

**Comment by fricklerhandwerk** [comment]
Here's a noncommittal sketch for reference

![image](https://github.com/user-attachments/assets/4ebb0242-3e37-4024-819a-b0ff7346ac64)

**Comment by erictapen** [comment]
@fricklerhandwerk and discussed this:

- The activity log is *not* to be shown in a detail view, but is displayed in a suggestion list (suggestions, selected, dismissed)
- It's placed in the upper right hand corner of the tile
- We assign three different colors to the categories **automaticly generated**, **dismissed** and **selected**. The exact color values are to be determined. The colors are supposed to match the colors on the state transition buttons. We need a CSS color palette for that.
- Timestamps are shown in relative times by default. When clicking on it, all the timestamps in one activity log turn into absolute timestamps.
- Bulk removal of packages is abbreviated. The list of removed (or added) packages is shown with a further click on it. TBD

![tmp 25ioYUgVBB](https://github.com/user-attachments/assets/e98c1aea-c121-4e68-ac34-7b47f61b8cfe)

**Comment by fricklerhandwerk** [comment]
Closed by #366
